planet/src/main/webapp/js/search-bar.js

```diff
@@ -12,29 +12,76 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const UP_ARROW = 38;
+const DOWN_ARROW = 40;
+const ENTER_KEY = 13;
```

Comment:
Where are these key codes defined?

Reply:
In the search-bar script as global constants. Should I put them inside the function or in a separate file?

Reply:
I assume you got these key codes from some standard and didn't come up with the values yourself? Where is that standard and can you directly reference the key values from there? That way, your keyboard navigation won't break if someday the key values change.

Reply:
I actually got the keycodes form this page: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
I just realized that it's inadvisable so I'll go for another simpler way: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key

Reply:
Done!